cmd/sidecar/main.go

```diff
@@ -40,17 +46,33 @@ func main() {
 		os.Exit(1)
 	}
 
-	configSecrets, err := secretsConfig.NewFromEnv(tokenFilePath)
+	configStorage, err := storageConfig.NewFromEnv()
 	if err != nil {
 		errLogger.Printf(err.Error())
 		os.Exit(1)
 	}
+	// Create new Storage
+	stor, err := storage.NewStorageHandler(*configStorage)
+
+	//if conjurSecretsDest == 1  &&  authn.Config.ContainerMode != "init" {
```

Comment:
Also here add "TODO"

Reply:
... suppose to be something like:
	conjurSecretsDest := os.Getenv("CONJUR_SECRETS_DESTINATION")
	if len(conjurSecretsDest) == 0 {
		conjurSecretsDest = "none"
 	} else {
                  // TODO: Validate value is either "none" or "k8s_secrets". Otherwise output an error
	}
	errLogger.Printf("CONJUR_SECRETS_DESTINATION: %s", conjurSecretsDest)

Reply:
the last line should be logged with `infoLogger`, not `errLogger`

Reply:
also, loading `os.Getenv("CONJUR_SECRETS_DESTINATION")` should be done in the config file, not in main

Reply:
updated but kept the logic flow with a detailed header because this logic will be moved to their respective files and most probably will be changed again